labs/lab-1-html-and-css/index.html

```diff
@@ -0,0 +1,71 @@
+<!doctype html>
+
+<html>
+
+<head>
+
+<link rel="stylesheet" href="stylesheet.css">
+<link href="https://fonts.googleapis.com/css?family=Open+Sans:800" rel="stylesheet">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+</head>
+
+<body>
+
+<div class="main">
+  <header>
+  <p>chirpus inc.</p>
+  </header>
+
+	  <nav class="menu">
+			<ul class="main-menu">
+				<li class="main-li">
+					<a href="home.html">Home</a>
+				</li>
+			<li class="main-li">
+					<a href="work.html">Work</a>
+				</li>
+				<li class="main-li">
+					<a href="about.html">About Us</a>
+				</li>
+				<li class="main-li">
+					<a href="contact.html">Contact</a>
+				</li>
+			<li class="hire-us">
+			  <a href="hireus.html">Hire Us</a>
+			</li>
+			</ul>
+	  </nav>
+  </div>
+
+	  <div class="main-content">
+		  <div class="content">
+			<p class="text-name">chirpus inc.</p>
+      <p class="text">Proin pellentesque utricies gravada. Vivamus consequat tempus
+			lectus vel tempus. Duis cursus prottitor quam at consectetur.
+			Nulla lobortis laoreet ligula, sed aliquet risus vestibulum
+      pellentesque. Sed vehicula ultrices ante, quis porttitor orci
+			pulvinar in. Pellentesque faucibus elementum pulvinar. </p>
+
+
+		  </div>
+
+		  <div class="circles">
+        <div class="circle-lrg"></div>
```

Comment:
C'mon, `circle-large` or `large-circle`. No one likes a `lrg`. Don't be a miser with characters. They're cheap, and you're probably only typing these class names twice total.